### PR TITLE
Update permissions.yml

### PR DESCRIPTION
**Pull request by MrFlameYT**
https://www.youtube.com/watch?v=K9oOni4Ye9s
Proof of Quake's Powers: 3:36

### CODE REVIEW

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -8797,6 +8797,32 @@ groups:
         - ch.alias.haste0
         - ch.alias.haste1
         - ch.alias.haste2
+    quake:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: too many spaces on line 8800

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -8797,6 +8797,32 @@ groups:
...
+        - ch.alias.jump0
+        - ch.alias.jump1
+        - ch.alias.jump2
+        - ch.alias.jump3
+        - ch.alias.jump4
+        - ch.alias.jump5
+        - ch.alias.jump6
+        - ch.alias.jump7
+        - ch.alias.jump8
+        - ch.alias.jump9
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: why so much jump?

Comment (MrFlameYT) [reply PermissionsEx/permissions.yml]: look at the proof

Comment (MrFlameYT) [reply PermissionsEx/permissions.yml]: proof of jump 9:
https://www.youtube.com/watch?v=K9oOni4Ye9s
2:51

Comment (MrFlameYT) [reply PermissionsEx/permissions.yml]: its at 3:00 sorry

**Pull request merged**